common.gradle

```diff
@@ -31,11 +31,15 @@ android {
         disable 'DataExtractionRules' // We don't want the dev apps to use backup
         disable 'IconMissingDensityFolder' // Using scaled down launcher icon is fine for dev-apps
     }
+
+    buildFeatures {
+        buildConfig true
```

Comment:
This matches the previous behaviour and is required due to how we manage/inject the Version of the SDK (and then consume it via the IMA/GMA extensions).